examples/test_minimal/README.md

```diff
@@ -0,0 +1,22 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Might be worth having some description of the purpose of the terraform in this directory. 

Also, structurally, maybe the `examples` directory is not the right place for this, since the examples are intended to be user facing.

Reply:
In this repo the test_minimal was created in an experimental phase and apparently the Terratest case can be run into the minimal folder and it would work. In other modules I "had" to create new folders as I saw the existing ones had to had few modifications in order to actually work with an init+apply. (For example valid VPC with valid CIDR etc). What we can discuss later is the necessity of having specific folders to be used by Terratest, or we may want to make the ones we have right now (e.g. examples/minimal) usable (where init+apply works).

Reply:
After our last conversation I agree that we'd prefer having a different folder for test examples. Reason being the client won't be creating a VPC for each module in a real world example. Having a README explain that they may create additional resources in order to make the examples work is enough clarification in my point of view. I will be committing changes (moving to a folder in root) later today here and in other repos that may need that. Thanks!!!